ruby-service.sql

```diff
@@ -0,0 +1,79 @@
+-- Drop previous versions of the tables if they they exist, in reverse order of foreign keys.
+
+DROP TABLE IF EXISTS Events;
+DROP TABLE IF EXISTS Trailer;
+DROP TABLE IF EXISTS Item;
+Drop TABLE IF EXISTS TrailerUser;
+DROP TABLE IF EXISTS TrailerItem;
+DROP TABLE IF EXISTS Users;
+
+-- Create the schema.
+CREATE TABLE Users (
+  username varchar(25) PRIMARY KEY NOT NULL,
+  password varchar(50) NOT NULL,
+	type varchar(10) NOT NULL
```

Comment:
It might be good to make an enum for `type`

Reply:
Good point. I'll add that.